lib/reference/properties.mjs

```diff
@@ -11,6 +11,40 @@ export const acceptCustomIdentsProperties = new Set([
 	'list-style-type',
 ]);
 
+/** @type {import('stylelint').ShorthandToResetToInitialProperty} */
```

Comment:
@ybiquitous do you know how to introduce a `type` and reference it in the same PR?